<?xml version="1.0" ?>
<launch>

  <arg name="sim"		default="false"/>
  <arg name="config"		default="aist"/>
  <arg name="manager"		default=""/>

  <!-- Start live camera -->
  <node name="live_camera"
	pkg="usb_cam" type="usb_cam_node" output="screen">
    <rosparam file="$(find aist_routines)/config/live_camera.yaml"/>
  </node>

  <!-- Launch ArUco and calibrator -->
  <include file="$(dirname)/include/camera_calibration.launch">
    <arg name="camera_names"	value="live_camera"/>
    <arg name="image_topics"	value="image_raw"/>
    <arg name="config_file"	value="$(dirname
				       )/../config/$(arg config).yaml"/>
    <arg name="manager"		value="$(arg manager)"/>
  </include>

  <!-- Start rviz -->
  <include file="$(find aist_bringup)/launch/include/rviz.launch">
    <arg name="moveit_config"	value="$(find aist_moveit_config)"/>
    <arg name="rviz_config"
  	 value="$(find aist_camera_calibration
  		)/launch/include/camera_calibration.rviz"/>
  </include>

</launch>
